### DISCUSSION TITLE
tasmota-sml-parser & Meter Definition script generator

### DISCUSSION THREAD

**Discussion by Mart-Muc** [arendst/Tasmota#17395] [Ideas]
pls add this to the smartmeter page:

saves a lot of time !!!!

https://tasmota-sml-parser.dicp.net/
© 2022 [Andreas Thienemann](https://github.com/ixs/tasmota-sml-parser/)

**Comment by Benno00** [comment]
14:47:14.705 : 77 01 0b 0a 01 41 50 41 01 04 e6 62 48 07 01 00 62 0a ff ff 72 62 01 65 02 39 b8 f6 74 
14:47:14.723 : 77 07 01 00 60 32 01 01 01 01 01 01 04 41 50 41 01 
14:47:14.747 : 77 07 01 00 60 01 00 ff 01 01 01 01 0b 0a 01 41 50 41 01 04 e6 62 48 01 
14:47:14.776 : 77 07 01 00 01 08 00 ff 65 00 1c 01 04 01 62 1e 52 03 69 00 00 00 00 00 00 1c ac 01 
14:47:15.675 : 77 07 01 00 02 08 00 ff 01 01 62 1e 52 03 69 00 00 00 00 00 00 00 00 01 01 01 63 82 12 00 76 05 06 ad 2a de 62 00 62 00 
14:47:15.705 : 77 01 0b 0a 01 41 50 41 01 04 e6 62 48 07 01 00 62 0a ff ff 72 62 01 65 02 39 b8 f7 74 
14:47:15.722 : 77 07 01 00 60 32 01 01 01 01 01 01 04 41 50 41 01 
14:47:15.748 : 77 07 01 00 60 01 00 ff 01 01 01 01 0b 0a 01 41 50 41 01 04 e6 62 48 01 
14:47:15.777 : 77 07 01 00 01 08 00 ff 65 00 1c 01 04 01 62 1e 52 03 69 00 00 00 00 00 00 1c ac 01 
14:47:16.674 : 77 07 01 00 02 08 00 ff 01 01 62 1e 52 03 69 00 00 00 00 00 00 00 00 01 01 01 63 36 c5 00 76 05 06 ad 2a e1 62 00 62 00 
14:47:16.704 : 77 01 0b 0a 01 41 50 41 01 04 e6 62 48 07 01 00 62 0a ff ff 72 62 01 65 02 39 b8 f8 74 
14:47:16.723 : 77 07 01 00 60 32 01 01 01 01 01 01 04 41 50 41 01 
14:47:16.747 : 77 07 01 00 60 01 00 ff 01 01 01 01 0b 0a 01 41 50 41 01 04 e6 62 48 01 
14:47:16.777 : 77 07 01 00 01 08 00 ff 65 00 1c 01 04 01 62 1e 52 03 69 00 00 00 00 00 00 1c ac 01 
14:47:17.675 : 77 07 01 00 02 08 00 ff 01 01 62 1e 52 03 69 00 00 00 00 00 00 00 00 01 01 01 63 4e 0f 00 76 05 06 ad 2a e4 62 00 62 00 
14:47:17.706 : 77 01 0b 0a 01 41 50 41 01 04 e6 62 48 07 01 00 62 0a ff ff 72 62 01 65 02 39 b8 f9 74 
14:47:17.724 : 77 07 01 00 60 32 01 01 01 01 01 01 04 41 50 41 01 
14:47:17.748 : 77 07 01 00 60 01 00 ff 01 01 01 01 0b 0a 01 41 50 41 01 04 e6 62 48 01 
14:47:17.778 : 77 07 01 00 01 08 00 ff 65 00 1c 01 04 01 62 1e 52 03 69 00 00 00 00 00 00 1c ac 01 
14:47:18.676 : 77 07 01 00 02 08 00 ff 01 01 62 1e 52 03 69 00 00 00 00 00 00 00 00 01 01 01 63 5a 73 00 76 05 06 ad 2a e7 62 00 62 00 
14:47:18.706 : 77 01 0b 0a 01 41 50 41 01 04 e6 62 48 07 01 00 62 0a ff ff 72 62 01 65 02 39 b8 fa 74 
14:47:18.724 : 77 07 01 00 60 32 01 01 01 01 01 01 04 41 50 41 01 
14:47:18.749 : 77 07 01 00 60 01 00 ff 01 01 01 01 0b 0a 01 41 50 41 01 04 e6 62 48 01 
14:47:18.778 : 77 07 01 00 01 08 00 ff 65 00 1c 01 04 01 62 1e 52 03 69 00 00 00 00 00 00 1c ac 01 
14:47:19.675 : 77 07 01 00 02 08 00 ff 01 01 62 1e 52 03 69 00 00 00 00 00 00 00 00 01 01 01 63 66 f7 00 76 05 06 ad 2a ea 62 00 62 00 
14:47:19.704 : 77 01 0b 0a 01 41 50 41 01 04 e6 62 48 07 01 00 62 0a ff ff 72 62 01 65 02 39 b8 fb 74 
14:47:19.723 : 77 07 01 00 60 32 01 01 01 01 01 01 04 41 50 41 01 
14:47:19.748 : 77 07 01 00 60 01 00 ff 01 01 01 01 0b 0a 01 41 50 41 01 04 e6 62 48 01 
14:47:19.777 : 77 07 01 00 01 08 00 ff 65 00 1c 01 04 01 62 1e 52 03 69 00 00 00 00 00 00 1c ac 01 
14:47:20.675 : 77 07 01 00 02 08 00 ff 01 01 62 1e 52 03 69 00 00 00 00 00 00 00 00 01 01 01 63 72 8b 00 76 05 06 ad 2a ed 62 00 62 00 
14:47:20.705 : 77 01 0b 0a 01 41 50 41 01 04 e6 62 48 07 01 00 62 0a ff ff 72 62 01 65 02 39 b8 fc 74 
14:47:20.723 : 77 07 01 00 60 32 01 01 01 01 01 01 04 41 50 41 01 
14:47:20.747 : 77 07 01 00 60 01 00 ff 01 01 01 01 0b 0a 01 41 50 41 01 04 e6 62 48 01 
14:47:20.777 : 77 07 01 00 01 08 00 ff 65 00 1c 01 04 01 62 1e 52 03 69 00 00 00 00 00 00 1c ac 01 
14:47:21.677 : 77 07 01 00 02 08 00 ff 01 01 62 1e 52 03 69 00 00 00 00 00 00 00 00 01 01 01 63 44 d0 00 76 05 06 ad 2a f0 62 00 62 00 
14:47:21.705 : 77 01 0b 0a 01 41 50 41 01 04 e6 62 48 07 01 00 62 0a ff ff 72 62 01 65 02 39 b8 fd 74 
14:47:21.722 : 77 07 01 00 60 32 01 01 01 01 01 01 04 41 50 41 01 
14:47:21.747 : 77 07 01 00 60 01 00 ff 01 01 01 01 0b 0a 01 41 50 41 01 04 e6 62 48 01 
14:47:21.777 : 77 07 01 00 01 08 00 ff 65 00 1c 01 04 01 62 1e 52 03 69 00 00 00 00 00 00 1c ac 01 
14:47:22.667 : 77 07 01 00 02 08 00 ff 01 01 62 1e 52 03 69 00 00 00 00 00 00 00 00 01 01 01 63 1b 8b 00 76 05 06 ad 2a f3 62 00 62 00 
14:47:22.698 : 77 01 0b 0a 01 41 50 41 01 04 e6 62 48 07 01 00 62 0a ff ff 72 62 01 65 02 39 b8 fe 74 
14:47:22.715 : 77 07 01 00 60 32 01 01 01 01 01 01 04 41 50 41 01 
14:47:22.740 : 77 07 01 00 60 01 00 ff 01 01 01 01 0b 0a 01 41 50 41 01 04 e6 62 48 01 
14:47:22.770 : 77 07 01 00 01 08 00 ff 65 00 1c 01 04 01 62 1e 52 03 69 00 00 00 00 00 00 1c ac 01 
14:47:23.674 : 77 07 01 00 02 08 00 ff 01 01 62 1e 52 03 69 00 00 00 00 00 00 00 00 01 01 01 63 11 ea 00 76 05 06 ad 2a f6 62 00 62 00 
14:47:23.703 : 77 01 0b 0a 01 41 50 41 01 04 e6 62 48 07 01 00 62 0a ff ff 72 62 01 65 02 39 b8 ff 74 
14:47:23.722 : 77 07 01 00 60 32 01 01 01 01 01 01 04 41 50 41 01 
14:47:23.747 : 77 07 01 00 60 01 00 ff 01 01 01 01 0b 0a 01 41 50 41 01 04 e6 62 48 01 
14:47:23.777 : 77 07 01 00 01 08 00 ff 65 00 1c 01 04 01 62 1e 52 03 69 00 00 00 00 00 00 1c ac 01 
14:47:24.676 : 77 07 01 00 02 08 00 ff 01 01 62 1e 52 03 69 00 00 00 00 00 00 00 00 01 01 01 63 33 73 00 76 05 06 ad 2a f9 62 00 62 00 
14:47:24.707 : 77 01 0b 0a 01 41 50 41 01 04 e6 62 48 07 01 00 62 0a ff ff 72 62 01 65 02 39 b9 00 74 
14:47:24.724 : 77 07 01 00 60 32 01 01 01 01 01 01 04 41 50 41 01 
14:47:24.748 : 77 07 01 00 60 01 00 ff 01 01 01 01 0b 0a 01 41 50 41 01 04 e6 62 48 01 
14:47:24.779 : 77 07 01 00 01 08 00 ff 65 00 1c 01 04 01 62 1e 52 03 69 00 00 00 00 00 00 1c ac 01 
14:47:25.675 : 77 07 01 00 02 08 00 ff 01 01 62 1e 52 03 69 00 00 00 00 00 00 00 00 01 01 01 63 ea 46 00 76 05 06 ad 2a fc 62 00 62 00 
14:47:25.706 : 77 01 0b 0a 01 41 50 41 01 04 e6 62 48 07 01 00 62 0a ff ff 72 62 01 65 02 39 b9 01 74 
14:47:25.724 : 77 07 01 00 60 32 01 01 01 01 01 01 04 41 50 41 01 
14:47:25.749 : 77 07 01 00 60 01 00 ff 01 01 01 01 0b 0a 01 41 50 41 01 04 e6 62 48 01 
14:47:25.777 : 77 07 01 00 01 08 00 ff 65 00 1c 01 04 01 62 1e 52 03 69 00 00 00 00 00 00 1c ac 01
